packaging/msi/templates/product.wxs

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0"?>
+
+<?if $(sys.BUILDARCH)="x86"?>
+    <?define Program_Files="ProgramFilesFolder"?>
+<?elseif $(sys.BUILDARCH)="x64"?>
+    <?define Program_Files="ProgramFiles64Folder"?>
+<?else?>
+    <?error Unsupported value of sys.BUILDARCH=$(sys.BUILDARCH)?>
+<?endif?>
+
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+
+   <Product Id="*" UpgradeCode="{{.UpgradeCode}}"
+            Name="{{.Product}}"
+            Version="{{.VersionOk}}"
+            Manufacturer="{{.Company}}"
+            Language="1033">
+
+      <Package InstallerVersion="200" Compressed="yes" Comments="Windows Installer Package" InstallScope="perMachine"/>
+
+      <Media Id="1" Cabinet="product.cab" EmbedCab="yes"/>
+
+      <Upgrade Id="{{.UpgradeCode}}">
```

Comment:
This is just a rule for forcing the upgrade of Torus right? Is there a way we can run a command at the same time?

When we upgrade an existing install we should also restart the daemon via `torus daemon stop`. Ideally we also tell them that after upgrading they will need to log into Torus again via the command prompt (or whichever shell they use).